core/domain/exp_jobs_one_off.py

```diff
@@ -57,6 +57,43 @@
 ])
 
 
+class RemoveDeprecatedExplorationModelFieldsOneOffJob(
+        jobs.BaseMapReduceOneOffJobManager):
+    """Job that sets skill_tags, default_skin, skin_customization fields
+    in ExplorationModels to None in order to remove it from the datastore.
+    """
+
+    @classmethod
+    def enqueue(cls, job_id, additional_job_params=None):
+        super(RemoveDeprecatedExplorationModelFieldsOneOffJob, cls).enqueue(
+            job_id, shard_count=64)
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [exp_models.ExplorationModel]
+
+    @staticmethod
+    def map(exp_model):
+        removed_deprecated_field = False
+        for deprecated_field in [
+                'skill_tags', 'default_skin', 'skin_customizations']:
+            if deprecated_field in exp_model._properties:  # pylint: disable=protected-access
+                del exp_model._properties[deprecated_field]  # pylint: disable=protected-access
+                if deprecated_field in exp_model._values:  # pylint: disable=protected-access
+                    del exp_model._values[deprecated_field]  # pylint: disable=protected-access
+                removed_deprecated_field = True
+        if removed_deprecated_field:
+            exp_model.commit('cid', 'Remove field', [])
```

Comment:
What is "cid"? You might also want to make the commit message more explicit.

Typically for automatic migrations like this we use an admin or bot name (please check with @vojtechjelinek for which one is typical here).

Reply:
Let's not use commit here, but let's do `exp_models.ExplorationModel.put_multi([exp_model])`

Reply:
I tried using `put_multi`, but it caused the mapreduce job to fail. Any idea why?